src/test/java/org/beanpod/switchboard/controller/StreamControllerTest.java

```diff
@@ -22,19 +22,19 @@
 class StreamControllerTest {
   private StreamController streamController;
 
-  @Mock private StreamDaoImpl channelService;
-  @Mock private StreamMapper channelMapper;
+  @Mock private StreamDaoImpl streamService;
+  @Mock private StreamMapper streamMapper;
 
   @BeforeEach
   public void setup() {
     MockitoAnnotations.initMocks(this);
-    streamController = new StreamController(channelService, channelMapper);
+    streamController = new StreamController(streamService, streamMapper);
   }
 
   @Test
   void testGetChannels() {
     // given
-    when(channelService.getStreams()).thenReturn(StreamFixture.getIdList());
+    when(streamService.getStreams()).thenReturn(StreamFixture.getIdList());
```

Comment:
All the test method names in this class are wrong too and reference channels instead of streams

Reply:
It's fixed